pages/partials/question.ejs

```diff
@@ -80,45 +83,49 @@
             </span>
           </div>
         </div>
+        <% } %>
       </div>
     </div>
   </form>
   <% } %>
 
   
   <% if (question.type != 'Freeform') { %>
-    <div class="panel panel-primary question-block">
-      <div class="panel-heading">
-        <% if (typeof instance_question_info == 'undefined') { %>
-        <h3 class="panel-title"><%= question.title %></h3>
+  <div class="panel panel-primary question-block">
+    <div class="panel-heading">
+      <% if (typeof instance_question_info == 'undefined') { %>
```

Comment:
I assume this is here so that the question can be viewed (in instructor mode) in the absence of an assessment. Correct?

(Would it be helpful to start commenting code with this sort of information?)

Reply:
Added code comments for this. Definitely agree that non-obvious things like this should be documented (or even better, come up with more natural solutions).